webpack/assets/javascripts/react_app/components/SearchBar/SearchBarHelpers.js

```diff
@@ -1,8 +1,7 @@
-import URI from 'urijs';
+import { getURI, changeQuery } from '../../common/urlHelpers';
 
 export const resolveSearchQuery = searchQuery => {
-  const uri = new URI(window.location.href);
+  const uri = getURI();
   const data = { ...uri.query(true), search: searchQuery.trim(), page: 1 };
-  uri.query(URI.buildQuery(data, true));
-  window.Turbolinks.visit(uri.toString());
+  changeQuery(uri, data);
```

Comment:
Do we have to call `getURI()` and pass it to the `changeQuery` every time we want to use it?
Would it make sense if `changeQuery` will call `getURI` instead of the consumer?

Reply:
Sometime consumers need to `getURI()` before calling `changeQuery` (manipulating data etc), so if we put it inside it will be called twice for no reason.

Reply:
What if `changeQuery` would be `changeQuery(data, uri = getURI())`?